front/app/modules/commercial/analytics/admin/containers/Visitors/index.tsx

```diff
@@ -1,28 +1,71 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import moment, { Moment } from 'moment';
 
 // hooks
 import useFeatureFlag from 'hooks/useFeatureFlag';
+import useAnalytics from 'api/analytics/useAnalytics';
+import { useIntl } from 'utils/cl-intl';
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
 
 // components
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, Text } from '@citizenlab/cl2-component-library';
 import ChartFilters from 'containers/Admin/dashboard/overview/ChartFilters';
 import Charts from './Charts';
+import Warning from 'components/UI/Warning';
 
 // utils
 import { getSensibleResolution } from 'containers/Admin/dashboard/overview/getSensibleResolution';
 
 // typings
 import { IResolution } from 'components/admin/ResolutionControl';
 import { IOption } from 'typings';
+import { Query, QuerySchema } from 'api/analytics/types';
+import { Response } from '../../components/VisitorsLanguageCard/useVisitorLanguages/typings';
+
+import messages from './messages';
+
+const query = (): Query => {
+  const localesCountQuery: QuerySchema = {
+    fact: 'visit',
+    aggregations: {
+      all: 'count',
+      'dimension_date_first_action.date': 'first',
+    },
+  };
+
+  return {
+    query: localesCountQuery,
+  };
+};
 
 const Visitors = () => {
   const [startAtMoment, setStartAtMoment] = useState<Moment | null | undefined>(
     undefined
   );
+  const [uniqueVisitorDataDate, setUniqueVisitorDataDate] = useState<
+    Moment | undefined
+  >(undefined);
   const [endAtMoment, setEndAtMoment] = useState<Moment | null>(moment());
   const [projectId, setProjectId] = useState<string | undefined>();
   const [resolution, setResolution] = useState<IResolution>('month');
+  const { formatMessage } = useIntl();
+  const { data: appConfig } = useAppConfiguration();
+  const { data: analytics } = useAnalytics<Response>(query());
+
+  useEffect(() => {
```

Comment:
Would be even nicer to solve this without `useEffect`, but you would have to wrap `Visitors` in another component and calculate the `uniqueVisitorDataDate` there, and pass it down to `Visitors` where you can initialize the `startAtMoment` with it